### PR TITLE
Guide/serverless llama

### PR DESCRIPTION
**Pull request by raksiv**

### CODE REVIEW

File: public/images/guides/serverless-llama/featured.png
Comment (davemooreuws) [inline public/images/guides/serverless-llama/featured.png:1]: reduce the size of the llama and move it no more than 65% down the canvas, so the description text doesn't overlap it on site.

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by HomelessDinosaur — DISMISSED** [review]

**Review by davemooreuws — DISMISSED** [review]

**Review by davemooreuws — DISMISSED** [review]

**Review by jyecusch — DISMISSED** [review]

File: docs/guides/python/serverless-llama.mdx [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+  # How services will be deployed by default, if you have other services not running models
+  # you can add them here too so they don't use the same configuration
+  default:
+    lambda:
+      # Set the memory to 6GB to handle the model, this automatically sets additional CPU allocation
+      memory: 6144
+      # Set a timeout of 30 seconds (this is the most API Gateway will wait for a response)
+      timeout: 30
+      # We add more storage to the lambda function, so it can store the model
+      ephemeral-storage: 1024
```
Comment (tjholm) [inline docs/guides/python/serverless-llama.mdx:235]: The model is embedded, so is this actually required?

**Review by davemooreuws — APPROVED** [review]

File: docs/guides/python/serverless-llama.mdx [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+  - API
+  - AI & Machine Learning
+languages:
+  - python
+image: /docs/images/guides/serverless-llama/banner.png
+image_alt: 'Serverless llama guide banner'
+featured:
+  image: /docs/images/guides/serverless-llama/featured.png
+  image_alt: 'Serverless llama guide featured image'
+published_at: 2024-11-25
```
Comment (davemooreuws) [inline docs/guides/python/serverless-llama.mdx:13]:
```suggestion
published_at: 2024-11-28
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

File: docs/guides/python/serverless-llama.mdx [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+When you're ready to deploy the project, we can create a new Nitric [stack file](/get-started/foundations/projects?lang=python#stack-files) which will target AWS:
+
+```bash
+nitric stack new dev aws
+```
+
+Update the stack file `nitric.dev.yaml` with the appropriate AWS region and memory allocation to handle the model.
+
+```yaml title:nitric.dev.yaml
+provider: nitric/aws@1.14.0
```
Comment (davemooreuws) [inline docs/guides/python/serverless-llama.mdx:223]:
```suggestion
provider: nitric/aws@1.16.0
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

**Review by HomelessDinosaur — DISMISSED** [review]

File: docs/guides/python/serverless-llama.mdx [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+  - API
+  - AI & Machine Learning
+languages:
+  - python
+image: /docs/images/guides/serverless-llama/banner.png
+image_alt: 'Serverless llama guide banner'
+featured:
+  image: /docs/images/guides/serverless-llama/featured.png
+  image_alt: 'Serverless llama guide featured image'
+published_at: 2024-11-28
```
Comment (davemooreuws) [inline docs/guides/python/serverless-llama.mdx:13]:
```suggestion
published_at: 2024-12-02
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

File: docs/guides/python/serverless-llama.mdx (hunk 1 above)
Comment (davemooreuws) [reply docs/guides/python/serverless-llama.mdx:235]: @raksiv can you take a look at this?

Comment (raksiv) [reply docs/guides/python/serverless-llama.mdx:235]: @tjholm without an increase, I experienced intermittent timeouts.

File: docs/guides/python/serverless-llama.mdx [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+  - API
+  - AI & Machine Learning
+languages:
+  - python
+image: /docs/images/guides/serverless-llama/banner.png
+image_alt: 'Serverless llama guide banner'
+featured:
+  image: /docs/images/guides/serverless-llama/featured.png
+  image_alt: 'Serverless llama guide featured image'
+published_at: 2024-12-02
```
Comment (davemooreuws) [inline docs/guides/python/serverless-llama.mdx:13]:
```suggestion
published_at: 2024-12-16
```

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

**Review by jyecusch — APPROVED** [review]

**Review by HomelessDinosaur — APPROVED** [review]

**Pull request merged**